### PR TITLE
Add torching support to the benchmark.

### PR DESCRIPTION
**Pull request by Raynos**
Part 1 of getting perf in. This adds a bunch more shinies
to the Makefile and adding torch support in the benchmark

r: @kriskowal @jcorbin @rf @shannili

### CODE REVIEW

**Comment by ShanniLi** [comment]
lgtm

File: node/benchmarks/Makefile [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+flame-torch-relay-bench:
+	node index.js --relay --torch relay --torchFile ./flame.html --torchTime 10 --torchType flame \
+		-- --relay --skipPing -s 4096,16384 -p 1000,10000,20000
+
+# torch-trace-bench:
+# 	node index.js --trace --relay --torch server --torchFile ./flame.raw \
+# 		-- --skipPing -s 4096
+
+create-flame:
+	stackcollapse-stap.pl ./flame.raw > ./flame.folded
```
Comment (jcorbin) [inline node/benchmarks/Makefile]:
```
.PHONY: view-flame

%.folded: %.raw
    stackcollapse-stap.pl $< >$@

%.svg: %.folded
    flamegrpah.pl $< >$@

view-flame: flame.svg
    open $<
```

Comment (jcorbin) [inline node/benchmarks/Makefile]: https://www.gnu.org/software/make/manual/html_node/Pattern-Rules.html

**Comment by Raynos** [comment]
Rather then improving the Makefile on this branch; I will iterate on it when I use it next to verify that the refactor supports the wanted process.

**Pull request merged**